.evg.yml

```
@@ -32,6 +32,8 @@ buildvariants:
     run_on: ubuntu1804-large
     tasks:
       - name: "build"
+      - name: "unittest"
+      - name: "integrationtest"
```

```suggestion
      - name: "test-unit"
      - name: "test-integration"
```
just a small naming suggestion with a couple small benefits:
- the test tasks get automatically grouped together by the UI if/when it sorts tasks lexicographically
- easier to filter down to the correctness-test tasks by prefix filtering

It might make sense to apply this same convention to the `tdvt` task name as well

---

Sounds good, I'll add the test prefix